modules/tinymce/src/core/main/ts/keyboard/KeyboardOverrides.ts

```diff
@@ -35,6 +36,7 @@ const setup = (editor: Editor): Cell<Text> => {
     EnterKey.setup(editor);
     SpaceKey.setup(editor);
     InputKeys.setup(editor);
+    TabKey.setup(editor);
```

Comment:
Not related to this PR, but I assume the intention here is to eventually move all these keyboard overrides over to the model? As atm none of them run for RTC.

Reply:
yes I'm expecting we will either do:
- an automatic registration based on either `PreInit` or `Init` events
- add a model method to register keyboard shortcuts (or perhaps even a complete "content wrapper element has been created, have fun" method)